server/src/main/java/org/opensearch/index/query/IntervalsSourceProvider.java

```diff
@@ -456,7 +484,11 @@ public String getWriteableName() {
 
         @Override
         public void writeTo(StreamOutput out) throws IOException {
-            out.writeBoolean(ordered);
+            if (out.getVersion().onOrAfter(Version.V_1_3_0)) {
```

Comment:
This is right change but fails the build [1]. Ideally, it should go to `1.x` and than `main` but because it is reversed, we need to use `Version.V_2_0_0` here, than do `Version.V_1_3_0` for `1.x` and propagate it back to `main` (sorry about that dance).

[1] https://ci.opensearch.org/logs/ci/workflow/OpenSearch_CI/PR_Checks/Gradle_Check/gradle_check_2380.log

Reply:
Ugh, I was afraid of this.  I will fix it.

Reply:
@reta the fails in the linked log are from the rest spec using the new parameters which has been fixed in latest commits.   I am not positive its node to node serialization.

Reply:
same, still xcontent in rest spec.  I am changing to 2x checks now.

Reply:
:+1: should be it